pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -665,6 +665,30 @@ def parse_args(args=None):
                               'https://developer.android.com/guide/'
                               'topics/manifest/'
                               'activity-element.html'))
+
+    ap.add_argument('--android-entrypoint', dest='android_entrypoint',
+                    default='org.kivy.android.PythonActivity',
+                    help='defines which java class sill be used for startup, usually a subclass of PythonActivity')
+    ap.add_argument('--android-apptheme', dest='android_apptheme',
```

Comment:
As with the android entrypoint, this seems okay (albeit even more niche), it's a very simple option and there's no simple alternative.